
package remote.wise.client.UserAuthenticationService;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.3.1-patch-01
 * Wed Jul 03 14:32:30 IST 2013
 * Generated source version: 2.3.1-patch-01
 * 
 */

@WebFault(name = "IOException", targetNamespace = "http://webservice.service.wise.remote/")
public class IOException_Exception extends Exception {
    public static final long serialVersionUID = 20130703143230L;
    
    private remote.wise.client.UserAuthenticationService.IOException ioException;

    public IOException_Exception() {
        super();
    }
    
    public IOException_Exception(String message) {
        super(message);
    }
    
    public IOException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public IOException_Exception(String message, remote.wise.client.UserAuthenticationService.IOException ioException) {
        super(message);
        this.ioException = ioException;
    }

    public IOException_Exception(String message, remote.wise.client.UserAuthenticationService.IOException ioException, Throwable cause) {
        super(message, cause);
        this.ioException = ioException;
    }

    public remote.wise.client.UserAuthenticationService.IOException getFaultInfo() {
        return this.ioException;
    }
}
